src/test/java/com/hydratereminder/command/CommandInvokerTest.java

```diff
@@ -83,9 +83,11 @@ public void shouldReturnNothingWhenIsNotHydrateCommand() {
         // given
         CommandExecuted commandToExecute = new CommandExecuted("aa", new String[]{"hydrate"});
 
-        // when and then
-        commandInvoker.invokeCommand(commandToExecute);
+        // when
+        final Object result = commandInvoker.invokeCommand(commandToExecute);
```

Comment:
Is this test working ? 
Return type of invokeCommand is void.

Reply:
i expected the tests to be run by git actions, my bad.
i'm used to python 😅